### PR TITLE
Updating a point fix for issues/3399

### PR DESCRIPTION
**Pull request by abchatra**
Changing serial parameter name to serialNumber https://github.com/Microsoft/pxt/issues/3399

### CODE REVIEW

File: docs/projects/rps-teams.md [hunk 1]
```diff
@@ -117,9 +117,9 @@ let player_index = 0
 let players: number[] = [0]
 let weapon = 0
 let found = false
-radio.onDataPacketReceived(({ receivedNumber, serial }) => {
+radio.onDataPacketReceived(({ receivedNumber, serialNumber }) => {
```
Comment (pelikhan) [inline docs/projects/rps-teams.md]: Does this compile?

Comment (abchatra) [reply docs/projects/rps-teams.md]: Nope needs serial: serialNumber

Comment (abchatra) [reply docs/projects/rps-teams.md]: I will fix

Comment (abchatra) [reply docs/projects/rps-teams.md]: Users will still have to rename their variable from the gear box. I am going to abandon this fix for the right fix from Richard.

**Pull request closed without merging**